manifests/init.pp

```diff
@@ -33,19 +35,19 @@
 #  Number of hours delay time before errors are generated in case downloads consistently fail.
 #
 # @param nosymlinks
-#  do not create serial number symlinks.
+#  Do not create serial number symlinks.
 #
 # @param inet6glue
 #  use Net::INET6Glue
 #
 # @param noerrors
-#  do not produce errors.
+#  Do not produce errors.
 #
 # @param nowarnings
-#  do not produce warnings.
+#  Do not produce warnings.
 #
 # @param http_proxy
-#  List of http proxy URLs.
+#  List of http proxy URLs. For example htpp
```

Comment:
Is it a list? stringified?

Missing the example as well.

Reply:
Added an example, 

Given this already has a type set I'll document to what the type is. If someone needs more than one proxy they can submit an issue but I think it's been like this a long time.